### PR TITLE
Feature/df 940 jwt config

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: alv-jwt-starter/src/main/java/ch/admin/seco/alv/shared/jwt/JWTFilter.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        if (StringUtils.hasText(jwt) && this.tokenProvider.validateToken(jwt)) {
+            Authentication authentication = this.tokenProvider.getAuthentication(jwt);
+            SecurityContextHolder.getContext().setAuthentication(authentication);
+        }
+        filterChain.doFilter(servletRequest, servletResponse);
+    }
+
+    private String resolveToken(HttpServletRequest request) {
+        String bearerToken = request.getHeader(AUTHORIZATION_HEADER);
+        if (StringUtils.hasText(bearerToken) && bearerToken.startsWith("Bearer ")) {
```
Comment (emina-str) [inline alv-jwt-starter/src/main/java/ch/admin/seco/alv/shared/jwt/JWTFilter.java]: maybe to add "Bearer " also as constant.

File: alv-jwt-starter/src/main/java/ch/admin/seco/alv/shared/jwt/TokenProvider.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TokenProvider {
```
Comment (birom) [inline alv-jwt-starter/src/main/java/ch/admin/seco/alv/shared/jwt/TokenProvider.java]: I find this name confusing. This class validates the token and converts a given token to Authentication object.

Comment (fabiangotzen) [reply alv-jwt-starter/src/main/java/ch/admin/seco/alv/shared/jwt/TokenProvider.java]: What would you suggest? TokenValidator?

Comment (birom) [reply alv-jwt-starter/src/main/java/ch/admin/seco/alv/shared/jwt/TokenProvider.java]: TokenToAuthenticationConverter

**Pull request merged**